cypress/e2e/e2e/v2.design_spec.js

```diff
@@ -0,0 +1,66 @@
+// type definitions for Cypress object "cy"
+/// <reference types="cypress" />
+/// <reference types="../../support" /> 
+
+import { expect } from "chai";
```

Comment:
@Abhishek-kumar09 chai is already bundled with cypress so its not necessary to import it nor do npm install at all: https://docs.cypress.io/guides/references/bundled-libraries#Chai

Reply:
Oh, mario, I was getting import problems during running this test, that's why. 
But yeah, I would take this PR https://github.com/layer5labs/meshery-extensions-packages/pull/85 as a motivation to wrap the assertions under cypress object

Reply:
Seems fine now. Thanks for pointing out